lib/manifold/partitioner.ex

```diff
@@ -31,7 +31,11 @@ defmodule Manifold.Partitioner do
     # Set optimal process flags
     Process.flag(:trap_exit, true)
     Process.flag(:message_queue_data, :off_heap)
-    {:ok, Tuple.duplicate(nil, partitions)}
+    workers = for _ <- 0..partitions do
+      {:ok, pid} = Worker.start_link()
```

Comment:
It is probably a fine change. But this no longer supports respawning a worker if it somehow crashes. If you want to make this change you should remove `:trap_exit` and the `:EXIT` match.

Reply:
fixed.